generators/server/templates/src/main/java/package/web/rest/dto/_ManagedUserDTO.java

```diff
@@ -2,13 +2,20 @@
 <% if (databaseType == 'mongodb' || databaseType == 'sql') { %>
 import java.time.ZonedDateTime;<% } %>
 
+import java.util.Set;
+
 import <%=packageName%>.domain.User;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 
 /**
  * A DTO extending the UserDTO, which is meant to be used in the user management UI.
  */
 public class ManagedUserDTO extends UserDTO {
 
+    public static final int PASSWORD_MIN_LENGTH = 5;
```

Comment:
@notflorian Yes, I changed it because the default user/user authentication didn't work with JWT.